scss/atoms/typography/_typography.scss

```diff
@@ -12,10 +12,10 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( max-width: $viewport-sm ) {
```

Comment:
@robbbz actually it has become quite common in breakpoints and viewpoint related classes/terms. Most css frameworks use -md -sm -xs -lg for grid too. So I guess it should be fine.
But I am open to opinions, whatever most people on the team agree to.